scripts.py

```diff
@@ -2156,7 +2157,7 @@ def sort_migrations(self, migrations):
         one.
         """
 
-        def compare_migrations(first, second):
+        def compare_migrations(first):
```

Comment:
In Python 3 you can't sort with a comparison method that takes two arguments and says which one is bigger. You have to take one argument and generate a sort key.